chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/configuration/ChaosMonkeyConfiguration.java

```diff
@@ -138,7 +142,8 @@ public ChaosMonkeyRequestScope chaosMonkeyRequestScope(
 
   @Bean
   public ChaosMonkeyScheduler scheduler(
-      @Nullable TaskScheduler scheduler, ChaosMonkeyRuntimeScope runtimeScope) {
+      @Nullable @Qualifier(CHAOS_MONKEY_TASK_SCHEDULER) TaskScheduler scheduler,
```

Comment:
Probably `@Nullable` is not necessary any more since we inject the bean always in method `chaosMonkeyTaskScheduler`.

Therefore I would suggest to remove it

```suggestion
      @Qualifier(CHAOS_MONKEY_TASK_SCHEDULER) TaskScheduler scheduler,
```

Reply:
yeah Nullable is not needed now, will remove it.

Reply:
Removed nullable.